src/std/main.ab

```diff
@@ -128,3 +128,52 @@ pub fun includes(arr, value) {
     }
     return false
 }
+
+pub fun create_symbolic_link(origin: Text, destination: Text): Bool {
+    if file_exist(origin) {
+        unsafe $ln -s "{origin}" "{path}"$
+        return true
+    }
+
+    echo "The file {$origin} doesn't exist!"
+    return false
+}
+
+pub fun create_dir(path: Text): Null {
+    if not dir_exist(origin) {
+        unsafe $mkdir -p "{path}"$
+    }
+}
+
+pub fun make_executable(path: Text): Bool {
+    if file_exist(origin) {
+        unsafe $chmod +x "{path}"$
+        return true
+    }
+
+    echo "The file {$path} doesn't exist!"
+    return false
+}
+
+pub fun make_executable(path: Text): Bool {
+    if file_exist(origin) {
+        unsafe $chmod +x "{path}"$
+        return true
+    }
+
+    echo "The file {$path} doesn't exist!"
+    return false
+}
+
+pub fun switch_user_permission(user: Text, path: Text): Bool {
+     if file_exist(origin) or dir_exist(origin) {
+        unsafe $chown -R "{user}" "{path}"$
+        return true
+    }
+
+    return false
+}
+
+pub fun download(url: Text, path: Text): Bool {
```

Comment:
There is a way to set a default value for a parameter in a function in Amber?
We can set `./` for `path`

Reply:
Not yet but it's on the list: https://github.com/Ph0enixKM/Amber/issues/124